app/forms/declaration/TransportCountry.scala

```diff
@@ -44,6 +46,9 @@ case class TransportCountry(countryName: Option[String]) extends Ordered[Transpo
 
   def value: String = countryName.getOrElse("")
 
+  def displayValue(countryHelper: CountryHelper)(implicit codeListConnector: CodeListConnector, messages: Messages): String =
```

Comment:
IMO this method does not belong here. I would move it in `view.helpers.CountryHelper` and call it from there in the view or where else need it.

Reply:
The call site is in PointerRecord for the various fields that have a cached country code.
I agree, it should be in the views package though.

Reply:
I could not find it because `PointerRecord`, being a big file, is by default not expanded by Github :slightly_smiling_face: